### DISCUSSION TITLE
remove config export button

### DISCUSSION THREAD

**Discussion by LoneStalkerz** [2dust/v2rayNG#2442] [General]
hi... i need to delete config export button from the screen... how can i do that?
i tried apk editor but i failed

**Comment by Amir-yazdanmanesh** [comment]
I can customize it for you, but my APK has a different signature compared to the app available on Google Play, so it cannot be installed on top of the existing app.

**Reply by hamed68persia** [reply]
salam khobi mohandes?man ye moshkeli daram azaton rahnamayi mikham mamnon misham ye shomare az telegram ya whatsapp ya har chizi ke betonam bahaton tamas begiram behem bedin.kheyli mamnon

**Comment by hamed68persia** [comment]
manam mikham gozine share to narm afzar v2ray hazf konam rahnamayi mikham

**Comment by hamed68persia** [comment]
ostad amir yazdan mansesh lotfan ye shomare telegrami ya whatsapp behem bedin bahaton ertebat dashte basham mamnonam
